README.md

```diff
@@ -1,2 +1,10 @@
-# SRepExtension
-Collection of Skeletal Model Extensions for 3D Slicer/SlicerSALT
+# Skeletal Representation Extension
+## Description
+Skeletal representation has been proved a robust model for shape analysis. This toolkit provides utilities to establish, refine and visualize skeletal representation model of an object.
+## Features
+This extension has 2 major features at the moment:
+1. Establish skeletal representation of an object
+2. Visualize the skeletal representation
+![alt text](https://github.com/ZhiyLiu/SRepExtension/blob/master/visualize_s_rep.png "Visualization of skeletal representation")
```

Comment:
Change accordingly for the new name of the file. I think we do not need to provide the full URL, we can use relative paths, so use:
![alt text](visualize_s_rep_NEW_NAME.png "Visualization of skeletal representation")